samples/logging/new-relic/Metrics_2/Endpoint/Endpoint.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>net461</TargetFrameworks>
+    <OutputType>Exe</OutputType>
+    <LangVersion>7.1</LangVersion>
+    <AutoGenerateBindingRedirects>false</AutoGenerateBindingRedirects>
```

Comment:
not sure why this is here, but I had to remove this from the v8 version because this was causing assembly loading issues due core's reference to System.Memory

Reply:
ugh, commented on the wrong PR 😆 ignore this.